src/codec/TlvObjectCodec.ts

```diff
@@ -20,6 +20,10 @@ export interface Field<T> extends TaggedTemplate<T> {optional: false};
 export interface OptionalField<T> extends TaggedTemplate<T> {optional: true};
 export type FieldTemplates = {[key: string]: Field<any> | OptionalField<any>};
 
+interface BitTemplate { position: number }
```

Comment:
The added Butmap support by fault or intent? (I have no issue adding it, but formally different topic ... we can be pragmatic :-)

Reply:
Oups, by fault. I will move to another PR